crates/net/src/sim2h_worker.rs

```diff
@@ -371,6 +336,7 @@ impl Sim2hWorker {
         Ok(())
     }
 
+    #[allow(dead_code)]
     fn send_ping(&mut self) {
```

Comment:
is send_ping not being used anywhere? Should we keep it?

Reply:
@AshantiMutinta Not sure if @lucksus intends to bring it back soon, otherwise I agree w/you.

Reply:
Yeah, I thought we might want to use it again. That's why I've marked it with this instead of removing it..